database/qb/insert.go

```diff
@@ -92,19 +66,33 @@ func (q *InsertQuery) ParameterizedSQL() (string, error) {
 		if col.Table != q.columns[0].Table {
 			return "", errors.New("insert columns must be from the same table")
 		}
-		qms[i] = ":" + col.GetName()
+		if parameterized {
+			qms[i] = ":" + col.GetName()
```

Comment:
qms is no longer a good variable name since it is holding more than question marks now. Maybe 'valuePlaces' or something.